core/src/main/java/io/grpc/CallOptions.java

```diff
@@ -328,12 +339,13 @@ private CallOptions() {
   }
 
   /**
-   * Returns whether the call is
-   * <a href="https://github.com/grpc/grpc/blob/master/doc/fail_fast.md">fail fast</a>.
-   * Fail fast is the default option for gRPC calls.
+   * Returns whether 'wait for ready' option is enabled for the call.
+   * <a href="https://github.com/grpc/grpc/blob/master/doc/fail_fast.md">'Fail fast'</a>
+   * is the default option for gRPC calls and 'wait for ready' is the opposite to it.
    */
-  public boolean isFailFast() {
-    return !waitForReady;
+  @ExperimentalApi("https://github.com/grpc/grpc-java/issues/1915")
+  public boolean isWaitForReady() {
```

Comment:
I agree that `isWaitForReady()` is also a bit awkward. But "should" or "will" are not a convention we use anywhere. I'm torn on it, but I think I prefer 'is' because it probably won't hurt anyone and will be more predictable as it follows our conventions. Meh.